website/versioned_docs/version-3.0/guides/no-suspense.md

```diff
@@ -10,49 +10,18 @@ In these cases you might want a boolean `loading` instead of yielding to Suspens
 In any case, here's a sample hook you can adapt to use that information in any
```

Comment:
As with @rest-hooks/test - this library works with 3.0 as well so let's update the recommendation